data/blog/js-sdk-v8-retrospective.md

```diff
@@ -0,0 +1,63 @@
+
+It is not possible to provide different docs for different versions of the SDK due to the way our docs work today. This meant that we were not able to write docs for certain v8 features before v8 actually shipped, which in turn also led to docs constantly lagging behind implementation a bit.
+
+We are planning to implement versioned docs in the future, which will allow us to start writing docs for an upcoming major while it is still in prerelease, ensuring that the docs are fully finished by the time we do a stable release of the SDK.
+
+### Update docs in tandem with deprecations
+
+We shipped a lot of deprecations in v7, and included replacement variants for all the things we deprecated. However, we often lagged behind in updating our docs to reflect the new way of doing things, leading to confusion with users. Going forward, we plan to update docs right after we ship a deprecation, ensuring the code snippets on docs never lead to deprecation warnings.
+
+### Be critical of adding deprecations
+
+We always try to be critical when adding deprecations to the SDK codebase. However, looking back, there are certainly a few places where we deprecated things that were not _strictly necessary_ to be deprecated. We should always ask ourselves a second and third time if a deprecation is truly required before adding it.
+
+### Be more active in the OpenTelemetry community
+
+Since we heavily rely on OpenTelemetry now, we want to be active parts of the community and contribute as much as possible. We have already started attending meetings and plan to expand on this even more in the future.
+
+## Things that went well
+
+There were also a lot of things that went well during the v8 release cycle:
+
+- We have good coverage of E2E test apps testing “real world” scenarios with the SDKs. This helped a lot with confidence building. Additionally, we also wrote some example apps to further test how things behaved.
+- The general process to implement deprecations with their replacements in v7 and then just removing the deprecated methods in v8 worked well, mostly.
+- Supporting backported fixes/features in v7 while working on v8 led to few issues. We ended up shipping quite a few v7 versions while working on v8, including a series of features and improvements.
+- The pre-release cycle (alpha, beta, rc) of v8 helped us to uncover a series of bugs, fulfilling its purpose.
+- After the release of v8, we worked hard on resolving issues that came up, and were actually able to resolve a series of problems (most of them coming from OpenTelemetry itself) in just a few weeks.
+
+## Onwards...
+
+Big shoutout to the whole team that made this massive release possible! It has been quite a ride, and it was only possible due to the combined effort of the JavaScript SDK team. Now, if you'll excuse us, we'll continue to iterate on the SDK (actually, we are [already at v8.18.0](https://github.com/getsentry/sentry-javascript/releases/tag/8.18.0)).
```

Comment:
i changed it to JS SDK team in the opening, cause no one know what web .... is :D 

and in the end you say the same, so i think it will make more sense